src/modules/nfts/components/wizard/PriceSummary.tsx

```diff
@@ -115,9 +123,16 @@ export default function PriceSummary({
             <Paragraph style={{ opacity: 0.6, fontSize: 14 }}>Total:</Paragraph>
             <Col>
               <Row>
-                <Paragraph style={{ fontSize: 18, marginBottom: 0 }}>
-                  <span className="sol-icon">◎</span> {totalSolCost}
-                </Paragraph>
+                <div
+                  style={{
+                    fontSize: 18,
+                    display: 'flex',
+                    alignItems: 'center',
+                    marginTop: '-0.25rem',
+                  }}
+                >
```

Comment:
Same on wrapping or add a class to the the symbol component and target with a styled component already wrapping the symbol.